src/librustdoc/html/render/mod.rs

```diff
@@ -1582,7 +1595,16 @@ fn render_impl(
             clean::AssocTypeItem(ref bounds, ref default) => {
                 let id = cx.derive_id(format!("{}.{}", item_type, name));
                 write!(w, "<h4 id=\"{}\" class=\"{}{}\"><code>", id, item_type, extra_class);
-                assoc_type(w, item, bounds, default.as_ref(), link.anchor(&id), "", cx.cache());
+                assoc_type(
+                    w,
+                    item,
+                    bounds,
+                    default.as_ref(),
+                    link.anchor(&id),
+                    "",
+                    cx.cache(),
+                    tcx,
+                );
```

Comment:
Rather than passing both of these, could you just pass a `cx`?

Reply:
Same: I can, but then I'd have to call again both `cx.cache()` and `cx.tcx()`. So as you prefer?

Reply:
Nah, this is fine.